packages/react-docs/RELEASE-NOTES.md

```diff
@@ -3,6 +3,124 @@ id: Release notes
+- **Page:**
+  - Removed redundant padding value in example ([#8485](https://github.com/patternfly/patternfly-react/pull/8485))
+  - Adds content to page component examples. ([#8198](https://github.com/patternfly/patternfly-react/pull/8198))
+- **Progress stepper:**
+  - Updated tests to new React testing library standards ([#8437](https://github.com/patternfly/patternfly-react/pull/8437))
+- **Search input:**
+  - Edited `ExpandableInput` and `SearchAttribute` for consistency ([#8527](https://github.com/patternfly/patternfly-react/pull/8527))
+  - Reordered props in onChange and onSearch ([#8516](https://github.com/patternfly/patternfly-react/pull/8516))
+  - Added a name prop to the input ([#8554](https://github.com/patternfly/patternfly-react/pull/8554))
+- **Select:**
+  - Added toggle ref ([#8528](https://github.com/patternfly/patternfly-react/pull/8528))
+- **Select next:**
+  - Updated logic for keyboard interaction ([#8496](https://github.com/patternfly/patternfly-react/pull/8496))
+- **Simple list:**
+  - Added aria label added to ungrouped simple list ([#8507](https://github.com/patternfly/patternfly-react/pull/8507))
+- **Slider:**
+  - Converted examples to TypeScript ([#8164](https://github.com/patternfly/patternfly-react/pull/8164))
+  - Updated input text styling ([#8392](https://github.com/patternfly/patternfly-react/pull/8392))
+- **Tabs:**
+  - Added event to `onAdd` callback ([#8492](https://github.com/patternfly/patternfly-react/pull/8492))
+  - Removed beta isHelpAction prop from TabsAction ([#8560](https://github.com/patternfly/patternfly-react/pull/8560))
+- **Text content:**
+  - Fixed test file name extension ([#8491](https://github.com/patternfly/patternfly-react/pull/8491))
+- **Text input group:**
+  - Reordered `onChange` parameters ([#8494](https://github.com/patternfly/patternfly-react/pull/8494))
+- **Text list:**
+  - Added `isPlain` prop ([#8562](https://github.com/patternfly/patternfly-react/pull/8562))
+- **Time picker:**
+  - Added event to `onChange` callback ([#8493](https://github.com/patternfly/patternfly-react/pull/8493))
+- **Toolbar:**
+  - Converted examples to typescript ([#8404](https://github.com/patternfly/patternfly-react/pull/8404))
+- **Wizard:**
+  - Fixed to put focus in drawer when opened in wizard ([#8540](https://github.com/patternfly/patternfly-react/pull/8540))
+- **Wizard next:**
+  - Recreated all legacy wizard examples ([#8422](https://github.com/patternfly/patternfly-react/pull/8422))
+
+### Access console extension
+- Introduced RFB clipViewport and dragViewport options ([#8470](https://github.com/patternfly/patternfly-react/pull/8470))
+- Removed react console extension from patternfly-react repo ([#8575](https://github.com/patternfly/patternfly-react/pull/8575))
+
+### Catalog view extension
+- Removed catalog view extension from patternfly-react repo ([#8474](https://github.com/patternfly/patternfly-react/pull/8474))
+
+### Log viewer:**
```

Comment:
```suggestion
### Log viewer:
```

Reply:
Looks like we still want to push this small change! Otherwise release notes look great 🙂